Framework/DataHandling/src/H5Util.cpp

```diff
@@ -201,7 +201,7 @@ std::string readString(H5::DataSet &dataset) {
 #if H5_VERSION_GE(1, 14, 0)
   // hdf5 >=1.14 puts the null terminator in the string
   // this strips that out
-  value.erase(std::find(value.begin(), value.end(), '\0'), value.end());
+  value.erase(value.rfind('\0'), 1);
```

Comment:
why is `count=1` necessary if character `\0` is supposed to be at at the end of string `value`? 🤔

Reply:
It was a guess that didn't work out